app/controllers/landable/api/access_tokens_controller.rb

```diff
@@ -42,6 +45,17 @@ def find_own_access_token(id = params[:id])
       def asset_token_params
         params.require(:access_token).permit(:username, :password)
       end
+
+      def determine_permissions(user_groups)
```

Comment:
Assignment Branch Condition size for determine_permissions is too high. [17.29/15]